docs/DeveloperGuide.md

```diff
@@ -330,23 +383,87 @@ testers are expected to do more *exploratory* testing.
+
+   1. Prerequisites: List all vendors using the `vendor` command.
 
-1. Deleting a person while all persons are being shown
+   1. Test case: `vendor i/1`<br>
+      Expected: First vendor is selected. The menu from the selected vendor is displayed.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Test case: `vendor`<br>
+      Expected: No vendor is selected. The list of vendors is displayed again.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `vendor i/0`<br>
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No vendor is selected. Error details shown in status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect delete commands to try: `vendor i/x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
 1. _{ more test cases …​ }_
 
+### Creating a new supper order
+
+1. Creating a supper order while a vendor has been chosen
+
+   1. Prerequisites: Select a vendor using the `vendor i/x` command (where x is the index of the chosen vendor).
+
+   1. Test case: `create`
+
+      Expected: New supper order created.
+
+1. _{ more test cases …​ }_
+
+### Viewing a current order
+
+1. Viewing an order while the order is currently active
+
+   1. Prerequisites: Create a supper order using the `view` command.
+
+   1. Test case: `order`<br>
+      Expected: All the items in the current order are displayed along with their associated quantities.
+
+1. _{ more test cases …​ }_
+
+### Adding an item
+
+1. Adding an item while an order is currently active
+
+   1. Prerequisites: Create a supper order using the `view` command.
+
+   1. Test case: `add i/1 q/1`<br>
+      Expected: 1 order of the first item from the menu is added into the order. Details of the added order shown in the status message. Timestamp in the status bar is updated.
+
+   1. Test case: `add i/1 q/0`<br>
+      Expected: No item is added. Error details shown in the status message. Status bar remains the same.
+
+   1. Test case: `add i/0 q/1`<br>
+
+      Expected: No item is added. Error details shown in the status message. Status bar remains the same.
+
+   1. Other incorrect delete commands to try: `add`, `add 0 9`, `add i/x q/y`, `...` (where x is larger than the menu size)<br>
+      Expected: Similar to previous.
+
+1. _{ more test cases …​ }_
+
+### Deleting an item
+
+1. Deleting an item while all items currently in the order are being shown
+
+   1. Prerequisites: List all items using the `view` command. Multiple items in the order.
```

Comment:
Should the second sentence be phrased as "There are at least two items in the order"